config/manager/kustomization.yaml

```diff
@@ -3,6 +3,7 @@ kind: Kustomization
 
 resources:
 - manager.yaml
+- namespace.yaml
```

Comment:
nit: we could directly place ns inside the manager.yaml same way as in https://github.com/kubernetes-sigs/cluster-api-ipam-provider-in-cluster/blob/main/config/manager/manager.yaml#L1-L6